feature-libs/my-account/organization/components/shared/organization-list/organization-list.component.html

```diff
@@ -25,6 +25,8 @@ <h3>
             </ng-option>
           </ng-select>
 
+          <ng-content></ng-content>
```

Comment:
I'd rather use the explicit projection that we used in the other shared components, i.e. in `organization-component.html`:

```html
<ng-content select="[actions]"></ng-content>
```